packages/esm-patient-tests-app/src/edit-test-results/modal/edit-lab-results.modal.tsx

```diff
@@ -0,0 +1,131 @@
+    const orderToEdit = orders.find((order) => order.uuid === selectedOrder);
+    if (orderToEdit) {
+      launchWorkspace('test-results-form-workspace', { order: orderToEdit });
+      closeModal();
+    }
+  };
+
+  const firstOrder = filteredOrders[0];
+
+  // Determine modal title and description based on number of orders
+  const modalTitle =
+    orders.length > 1 ? t('selectTestToEdit', 'Select test to edit') : t('editLabResults', 'Edit lab results');
+
+  const confirmationText =
+    orders.length > 1
+      ? t('selectTestPrompt', 'Please select which test you would like to edit for the following patient:')
+      : t('confirmationText', 'Do you want to edit {{test}} results for the following patient?', {
+          test: orders?.[0]?.concept?.display,
+        });
+
+  return (
+    <>
+      <ModalHeader closeModal={closeModal} title={modalTitle} />
+      <ModalBody>
+        <p className={styles.titleHeader}>{confirmationText}</p>
+        <div className={classNames(styles.modalBody, styles.modalContentWrapper)}>
+          {/* Patient Details Panel - Now on the left */}
+          <div className={styles.patientPanel}>
+            <h4 className={styles.sectionHeader}>{t('patientDetails', 'Patient Details')}</h4>
+            <div className={styles.patientContent}>
+              {filteredOrders.length > 0 && (
+                <div className={styles.patientInfo}>
+                  <div className={styles.patientDetailsSection}>
+                    <p className={styles.itemLabel}>
+                      <span className={styles.labelKey}>{t('name', 'Name')}:</span>
+                      <span className={styles.labelValue}>{firstOrder?.patient?.person?.display}</span>
+                    </p>
+                    <p className={styles.itemLabel}>
+                      <span className={styles.labelKey}>{t('age', 'Age')}:</span>
+                      <span className={styles.labelValue}>{firstOrder?.patient?.person?.age}</span>
+                    </p>
+                    <p className={styles.itemLabel}>
+                      <span className={styles.labelKey}>{t('gender', 'Gender')}:</span>
+                      <span className={styles.labelValue}>
+                        {firstOrder?.patient?.person?.gender === 'M' ? t('male', 'Male') : t('female', 'Female')}
+                      </span>
+                    </p>
+                  </div>
+
+                  <div className={styles.encounterDetails}>
+                    <p className={styles.itemLabel}>
+                      <span className={styles.labelKey}>{t('location', 'Location')}:</span>
+                      <span className={styles.labelValue}>{location}</span>
+                    </p>
+                    <p className={styles.itemLabel}>
+                      <span className={styles.labelKey}>{t('dateOrdered', 'Date Ordered')}:</span>
+                      <span className={styles.labelValue}>{formatDate(new Date(firstOrder.dateActivated))}</span>
```

Comment:
@its-kios09 this no longer works if a patient has more than one completed lab. The error comes from this line because the `dateActivated` is already formatted
<img width="1697" alt="Screenshot 2025-06-18 at 12 37 03 PM" src="https://github.com/user-attachments/assets/396115ac-f83b-4e07-9664-02ba6aa3b85f" />